db/migrate/20250211192052_add_event_type_to_user_action_events.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class AddEventTypeToUserActionEvents < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def up
+    # Remove old slug column if it exists
+    if column_exists?(:user_action_events, :slug)
+      remove_index :user_action_events, :slug, algorithm: :concurrently if index_exists?(:user_action_events, :slug)
+      safety_assured { remove_column :user_action_events, :slug }
+    end
```

Comment:
this doesn't exist in production so we don't need this. 
FYI If you ever need to reverse migrations to change a column name or something you can just do 

```bash
rails db:migrate:down VERSION=20250211192052
```
-make changes to the migration file-

```bash
rails db:migrate:up VERSION=20250211192052 # or just rails db:migrate
```